sourcecode/hub/resources/views/user/reset-password.blade.php

```diff
@@ -0,0 +1,31 @@
+<x-layout>
+    <x-slot:title>{{ trans('messages.reset-password') }}</x-slot:title>
+
+    <x-form action="{{ route('reset-password-update', ['token' => $token, 'email' => $email]) }}">
+        <x-form.field
```

Comment:
Since this field is entirely for display purposes, it shouldn't have a `name` attribute. Having one causes the email to be sent twice, as the email is also contained in the URL.